# rad
Accl_Limit_Params:
  MPC:
    MAX_JOINT_SPEED: 2.8
    MAX_JOINT_ACCL: 4.6
    MAX_JOINT_JERK: 50
    MAX_JOINT_SNAP: 200
  ACCL_LIMIT:
    MAX_JOINT_SPEED: 2.5
    MAX_JOINT_ACCL: 3.0

Control_Tasks:
  Control_Neck: true
  Control_Arm: true
  Control_Hand: true
  Control_Waist: false
Feedback_Tasks:
  Test_Send_Image: false
  Send_Joint_Feedback: true
  Send_Image_Feedback: false
  Send_Force_Feedback: false
Send_Target_Use_Planning: true
